src/main/java/rx/internal/util/BackpressureDrainManager.java

```diff
@@ -62,10 +62,7 @@
     }
 
     /** The request counter, updated via REQUESTED_COUNTER. */
-    protected volatile long requestedCount;
-    /** Atomically updates the the requestedCount field. */ 
-    protected static final AtomicLongFieldUpdater<BackpressureDrainManager> REQUESTED_COUNT
-    = AtomicLongFieldUpdater.newUpdater(BackpressureDrainManager.class, "requestedCount");
+    protected AtomicLong requestedCount = new AtomicLong();
```

Comment:
This should be either `final` or `BackpressureDrainManager` should extend `AtomicLong`.